src/main/java/frc/robot/subsystems/swerve/SwerveSubsystem.java

```diff
@@ -404,59 +454,72 @@ public Pose2d getFuturePose() {
    * @param time Time in the future to point from
    * @return A command refrence that rotates the robot to a computed rotation
    */
-  public Command pointTowardsTranslation(
+  public Command pointTowardsTranslationCmd(
       DoubleSupplier xMetersPerSecond, DoubleSupplier yMetersPerSecond, double time) {
     ProfiledPIDController headingController =
-        // assume we can accelerate to max in 0.5 seconds
+        // assume we can accelerate to max in 0.3 seconds
         new ProfiledPIDController(
-            1.0, 0.0, 0.0, new Constraints(MAX_ANGULAR_SPEED, MAX_ANGULAR_SPEED / 0.5));
+            40.0, 0.0, 0.0, new Constraints(MAX_ANGULAR_SPEED, MAX_ANGULAR_SPEED / 0.666666));
     headingController.enableContinuousInput(-Math.PI, Math.PI);
 
-    return this.runVelocityFieldRelative(
+    return Commands.sequence(
+        Commands.runOnce(
```

Comment:
this could be in the `beforeStarting` although thats somewhat confusing since it becomes out of order

Reply:
could move everything in `beforeStarting` here